CMakeLists.txt

```diff
@@ -18,10 +19,15 @@ add_library(convert_util SHARED convert_util.c)
 add_library(convert_client SHARED convert_client.c lib/log.c/src/log.c)
 target_link_libraries(convert_client PRIVATE syscall_intercept_static convert_util)
 
+# Unit & integration tests
+add_subdirectory(tests)
+
 # Tests for code linting & formatting
 enable_testing()
 
-set(LIBCONVERT_SOURCES convert_client.c convert.h convert_util.c convert_util.h)
+list(APPEND LIBCONVERT_SOURCES convert_client.c convert.h convert_util.c convert_util.h)
+
+message("${LIBCONVERT_SOURCES}")
```

Comment:
(do you need this? Or prepend with an explanation)

Reply:
No I don't need it! This was added for debugging, I can remove it now :-)